sources/packages/api/src/route-controllers/student-appeal/student-appeal.aest.controller.ts

```diff
@@ -135,4 +141,30 @@ export class StudentAppealAESTController extends BaseController {
       throw error;
     }
   }
+
+  /**
+   * Gets all pending student application appeals.
+   * @param pagination options to execute the pagination.
+   * @returns list of pending student application appeals.
+   */
+  @Get()
+  async getPendingAppeals(
```

Comment:
Pending seems to be a filter that would be applied to this `GET`. In the future, if we need to show the list of all Appeals I guess that this would be the route that we would be using.
Maybe we can either create a specific endpoint for "Pendings" (e.g. @Get("pending")`) or leave this for the future if such a requirement comes.
@ann-aot @guru-aot @dheepak-aot thoughts?

Reply:
I feel, we should create a specific endpoint. /pending

Reply:
Fixed

Reply:
I agree with @andrewsignori-aot . The endpoint can be more generic like getAppeals() and we can apply a query param to filter it. or getAppealsByStatus().